lang/en.json

```diff
@@ -165,8 +221,12 @@
 	"CYPHERSYSTEM.IotumAndParts": "Iotum & Parts",
 
 	"CYPHERSYSTEM.Modifier": "Modifier",
-	"CYPHERSYSTEM.EasedBy": "eased by",
-	"CYPHERSYSTEM.HinderedBy": "hindered by",
+	"CYPHERSYSTEM.Eased": "Eased",
+	"CYPHERSYSTEM.EasedBy": "Eased by",
```

Comment:
Is there an instance where “Eased by” and “Hindered by” are capitalized?

Reply:
No, you right, only "Eased" and "Hindered" are. Let me fix it.

Reply:
Refactored translation